source/tutorials/kubernetes.rst

```diff
@@ -3,354 +3,657 @@
+Background
 ***********
 
-Kubernetes is an open source system for automating deployment, scaling, and
-management of containerized applications. It groups containers that make up
-an application into logical units for easy management and discovery. Get up
-and running quickly with our `Cloud native setup automation`_.
+|CL| has builtin integrations to make setting up Kubernetes using a variety of
+`container runtimes
+<https://kubernetes.io/docs/setup/production-environment/container-runtimes/>`_.
+
+For more background information see:
+
+* `What is Kubernetes?`_
+* `What is a Container Network Interface (CNI)?`_
+
+* `What is a Container Runtime Interface (CRI)?`_ 
+
+  * `What is Docker?`_
 
-Kata Containers\* kata-runtime adheres to
-:abbr:`OCI (Open Container Initiative*)` guidelines and works seamlessly with
-Kubernetes. `Kata Containers`_ provides strong isolation for untrusted
-workloads or  multi-tenant scenarios. Kata Containers can be
-allocated on a per-pod basis, so you can mix and match both on the same host
-to suit your needs.
+  * `What is containerd?`_
+
+  * `What is CRI+O?`_
+
+* `What is Kata Containers?`_
 
 Prerequisites
 *************
 
 This tutorial assumes you have already installed |CL|. For detailed
-instructions on installing |CL| on a bare metal system, follow the
-:ref:`bare metal installation tutorial<bare-metal-install-desktop>`.
-Learn about the benefits of having an up-to-date system for cloud
-orchestration on the :ref:`swupd-guide` page.
+instructions on installing |CL| on a bare metal system, follow the :ref:`bare
+metal installation tutorial<bare-metal-install-desktop>`.
 
-Before you install any new packages, update |CL| with the following command:
+#. Before you continue, update your |CL| installation with the following command:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   sudo swupd update
+      sudo swupd update
 
-Install Kubernetes and CRI runtimes
-***********************************
+   Learn about the benefits of having an up-to-date system for cloud
+   orchestration on the :ref:`swupd-guide` page.
 
-Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
-runtimes, and networking plugins, are included in the `cloud-native-basic`_
-bundle.
+#. Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
+   runtimes, :abbr:`CNI (Container Network Interface)` and `cloud-native-setup
+   scripts`_ are included in the `cloud-native-basic`_ bundle. Install the
+   cloud-native-basic bundle to get these components:
 
-To install this framework, enter the following command:
+   .. code-block:: bash
 
-.. code-block:: bash
+      sudo swupd bundle-add cloud-native-basic
 
-   sudo swupd bundle-add cloud-native-basic
+
+Set up Kubernetes automatically
+*******************************
+
+|CL| provides `cloud-native-setup scripts`_ to automate system setup and
+Kubernetes cluster initialization which allows you to get a cluster up and
+running quickly.
 
 .. note::
```

Comment:
This did not render properly in my view.

Reply:
You have to build the docs locally to the documentation render properly. The GH render is a best-effort.